amundsen-kube-helm/templates/helm/amundsen/templates/deployment.yaml

```diff
@@ -144,6 +144,10 @@ spec:
         ports:
           - containerPort: 5000
         env:
+          {{ if .Values.FRONTEND_BASE }}
```

Comment:
should I add a comment in this file for why this is being set?

Reply:
that will be great!

Reply:
done!